docs/superpowers/plans/2026-06-24-gear-set-dot-pair.md

```
@@ -0,0 +1,558 @@
+- Modify: `src/constants/changelog.ts` (`UNRELEASED_CHANGES`)
+- Modify: `src/pages/DocumentationPage.tsx` (gear-set / DoT section)
+
+- [ ] **Step 1: Editor channel option.** Add to `MODIFIER_CHANNEL_OPTIONS` (after `outgoingDamage`):
+
+```typescript
+    { value: 'dotDamage', label: 'DoT Damage' },
+```
+
+- [ ] **Step 2: Changelog.** Add to `UNRELEASED_CHANGES` (plain English, user-facing):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)